templates/addons/etcd-backup-restore/etcd-backup-restore.yaml

```diff
@@ -0,0 +1,191 @@
+---
+apiVersion: v1
+kind: Secret
+type: addons.cluster.x-k8s.io/resource-set
+metadata:
+  name: ${CLUSTER_NAME}-ssekey
+stringData:
+  linode-token-region.yaml: |-
+    kind: Secret
+    apiVersion: v1
+    metadata:
+      name: sse-key
+      namespace: kube-system
+    stringData:
+      sse-key: ${SSE_KEY}
+---
+apiVersion: addons.cluster.x-k8s.io/v1beta1
+kind: ClusterResourceSet
+metadata:
+  name: ${CLUSTER_NAME}-ssekey-crs
+spec:
+  clusterSelector:
+    matchLabels:
+      etcd-backup: "true"
+      cluster: ${CLUSTER_NAME}
+  resources:
+  - kind: Secret
+    name: ${CLUSTER_NAME}-ssekey
+  strategy: ApplyOnce
+---
```

Comment:
can we break this linodeObjectStorageBucket into it's own file in the etcd-backup-restore flavor?

Reply:
I have broken this up into 3. OBJ, SSE key, and the etcdbr itself. Each also has its own ClusterResourceSet in the same file